apps/modernization-ui/src/apps/page-builder/context/DragDropProvider.tsx

```diff
@@ -301,9 +312,9 @@ const DragDropProvider: React.FC<{ children: React.ReactNode; data: PagesTab | u
         } else if (type === 'question') {
             handleQuestionMove(source, destination);
         } else {
-            // else its a Subsection move so we go here
             handleSubsectionMove(source, destination);
         }
+        reorderObjects(token, Number(destination.droppableId) + (destination.index - 1), moveId, pageDropId);
```

Comment:
Question (blocking): Passing `Number(destination.droppableId) + (destination.index - 1)` as the `after` component Id seems wrong. I haven't had my hands on the reorder UI side, but it seems that we would want to get the new index of the drop location.

If thats > 0, get the Id of component at new index - 1.
If drop index == 0, then we want the parent container Id.

If I'm misunderstanding something I may need you to give me a walkthrough of whats going on. I can be easily confused.